desafio-conceitos-nodejs/README.en.md

```diff
@@ -1,10 +1,10 @@
-<img alt="GoStack" src="https://storage.googleapis.com/golden-wind/bootcamp-gostack/header-desafios.png" />
+<img alt="GoStack" src="https://storage.googleapis.com/golden-wind/bootcamp-gostack/header-desafios.png" style="width: 100%;"/>
 
 <h3 align="center">
   Challenge 02: Node.js Concepts
 </h3>
 
-<p align="center">“Don't wait to to plant, only have patient to harvest”!</blockquote>
+<p align="center">“Don't wait to plant, only have patient to harvest”!</blockquote>
```

Comment:
Change patient for patience

Reply:
Good catch! It should be a subjective instead of a adjetive